gladier_tools/tests/posix/conftest.py

```diff
@@ -1,3 +1,3 @@
 import pytest
 import tarfile
-
+from test_tar import mock_tar
```

Comment:
Instead of importing it, let's place the mock_tar fixture directly in the conftest directory. That way, it will apply to both. 

That seems to be what's causing the error in the automated tests: https://github.com/globus-gladier/gladier-tools/runs/3166722951?check_suite_focus=true

Reply:
Woops, you're right. `conftest.py` _file_, not directory. My mistake!